Content.Server/HeightAdjust/BloodstreamAdjustSystem.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Server.Body.Components;
+using Content.Server.Chemistry.Containers.EntitySystems;
+using Content.Shared.CCVar;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Contests;
+using Content.Shared.HeightAdjust;
+using Robust.Shared.Configuration;
+
+namespace Content.Server.HeightAdjust;
+
+public sealed class BloodstreamAdjustSystem : EntitySystem
+{
+    [Dependency] private readonly IConfigurationManager _config = default!;
+    [Dependency] private readonly ContestsSystem _contests = default!;
+    [Dependency] private readonly SolutionContainerSystem _solutionContainer = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<BloodstreamAffectedByMassComponent, MapInitEvent>((uid, comp, _) => TryAdjustBloodstream((uid, comp)));
+        SubscribeLocalEvent<BloodstreamAffectedByMassComponent, HeightAdjustedEvent>((uid, comp, _) => TryAdjustBloodstream((uid, comp)));
+    }
+
+    /// <summary>
+    ///     Adjusts the bloodstream of the specified entity based on the settings provided by the component.
+    /// </summary>
+    public bool TryAdjustBloodstream(Entity<BloodstreamAffectedByMassComponent> ent)
+    {
+        if (!TryComp<BloodstreamComponent>(ent, out var bloodstream)
+            || !_solutionContainer.TryGetSolution(ent.Owner, bloodstream.BloodSolutionName, out var bloodSolutionEnt)
+            || !_config.GetCVar(CCVars.HeightAdjustModifiesBloodstream))
+            return false;
+
+        var bloodSolution = bloodSolutionEnt.Value.Comp.Solution;
+
+        var factor = Math.Pow(_contests.MassContest(ent, bypassClamp: true, rangeFactor: 4f), ent.Comp.Power);
```

Comment:
```suggestion
        var factor = Math.Pow(_contests.MassContest(ent, bypassClamp: true), ent.Comp.Power);
```
When bypassClamp is set to true, rangeFactor becomes irrelevant, since bypassClamp prevents Range from being a variable, making it so that a given Contest will return (-inf, inf) as its range. This'll be limited by https://github.com/Simple-Station/Einstein-Engines/pull/814 to instead return (Epsilon, 32bit Int Limit).

Reply:
It is still relevant in case some genius decides to set AllowClampOverride to true.